### PR TITLE
Support CentOS 7

### LINKED ISSUE #15: Problem: No CentOS/RedHat support
This role doesn't work on CentOS/RedHat

### PR DESCRIPTION
**Pull request by mamedin**
Percona 5.5 is installed and configured in the same way as in Ubuntu.

It Fixes #15

### CODE REVIEW

**Comment by mamedin** [comment]
When testing with `ansible-archivematica-src` role, take into account that `archivematicaCommon` and  `Storage Service` depend on `mariadb-devel `package, so you should manually add an `ignore_errors` to these osdeps tasks.

File: tasks/configure-centos.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+
+- name: "Create language dir symlink"
+  file:
+    src: /usr/share/percona-server
```
Comment (scollazo) [inline tasks/configure-centos.yml]: Parameters should be enclosed with quotes

File: tasks/configure-centos.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    dest: /usr/share/mysql
+    state: link
+  become: yes
+
+- name: "Update the my.cnf"
+  template: "src=etc_my.cnf-centos.j2 dest=/etc/my.cnf owner=root mode=0600"
```
Comment (scollazo) [inline tasks/configure-centos.yml]: For consistency,  each parameter should be in its own line

File: tasks/main.yml [hunk 3]
```diff
@@ -1,8 +1,16 @@
 ---
 - include: install.yml
   tags: install
+  when: ansible_os_family != "RedHat"
+- include: install-centos.yml
```
Comment (scollazo) [inline tasks/main.yml]: Steps should have a blank line between them

**Comment by hakamine** [comment]
@mamedin looking. I will rebase and add some extra travis config

**Pull request merged**